### ISSUE TITLE
Validation error when exporting element from file.

### ISSUE THREAD

**Issue by Aphrodites1995** [uxbox/uxbox#2009]
**To Reproduce**

1. Create any file
2. Add any element
3. Try to export it by clicking the add button, then Export 1 element

**Expected behavior**

The element is exported

**Actual behavior**

There is a red popup at the top right saying "Validation Error", and the docker containers return

```
penpot-penpot-exporter-1  | Error: connect ETIMEDOUT
penpot-penpot-exporter-1  |     at Socket.<anonymous> (/opt/app/node_modules/ioredis/built/redis/index.js:327:37)
penpot-penpot-exporter-1  |     at Object.onceWrapper (node:events:645:28)
penpot-penpot-exporter-1  |     at Socket.emit (node:events:526:28)
penpot-penpot-exporter-1  |     at Socket._onTimeout (node:net:501:8)
penpot-penpot-exporter-1  |     at listOnTimeout (node:internal/timers:559:17)
penpot-penpot-exporter-1  |     at processTimers (node:internal/timers:502:7)
penpot-penpot-exporter-1  | SEV [app.redis] hint="error on redis connection"
penpot-penpot-exporter-1  | WRN [app.redis] hint="connection closed"
```
on the command line.

**Screenshots**
<img width="1675" alt="Screen Shot 2022-06-18 at 23 10 14" src="https://user-images.githubusercontent.com/36265241/174444755-639e9bdb-ab8a-4e9f-8e75-6ba3ce390dfe.png">

**Desktop (please complete the following information):**
- OS (e.g. iOS): MacOS Monterey
- Browser & version (e.g. Chrome 89.0): Firefox Developer Edition, 102.0b8

**Environment (please complete the following information):**
- Host (e.g. https://design.penpot.app, local instance):
I have a local instance running on a docker container on localhost using the beginners guide and docker-compose.
I am using the Github release version penpot-1.13.0-beta

*If self-hosted:*
- OS Version (e.g. Ubuntu 16.04): MacOS Monterey
- Docker / Docker-compose version (e.g. Docker version 18.03.0-ce, build 0520e24): Docker version 20.10.16, build aa7e414
- Image version (e.g. Alpine): Dunno. Whatever this: `docker compose -p penpot -f docker-compose.yaml up -d` gives me

Docker commands or docker-compose file (if possible and if proceed.x):
All config.envs, docker-compose files, and others are consistent with the release.

Exporting the actual penpot file works, but exporting from the hamburger menu gives the same error.

**Comment by niwinz** [comment]
With the exporter logs, looks like is something misconfigured. Without more context, is very complicated to help.
Some hints to look if the public url is set correctly on exporter and on backend. If exporter raises timeout means it does not be able to connect to the frontend container.

**Comment by Pipazoul** [comment]
I have the same problem with the following logs

``` bash 
Error: getaddrinfo ENOTFOUND redis
penpot-exporter_1  |     at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:71:26)
penpot-exporter_1  | SEV [app.redis] hint="error on redis connection"
penpot-exporter_1  | WRN [app.redis] hint="connection closed"

```

The only param who differs from the initial docker compose file is the port mapping on the frontend service that i changed
is there a ENV var to set for the exporter uri ?

For now i'm using the 1.12.4-beta version who works like a charm :)

**Comment by niwinz** [comment]
The exporter on 1.13 requires connection to redis, ensure you have the docker-compose.yml updated, maybe you have missed some update:
![image](https://user-images.githubusercontent.com/843689/175023806-7a4cbd77-b8ee-4846-b0a7-8b47bb8a591d.png)

**Comment by dploeger** [comment]
@niwinz Yes, this is one problem.

The other one occurs after you change that. I get this now:

    {"~:explain":null,"~:type":"~:validation","~:hint":"looks like the uri provided is not part of the white list","~:code":"~:domain-not-allowed"}

I believe it has something to do with the public URI but I couldn't fix it.

(or should I open another issue for this?)

**Comment by niwinz** [comment]
Hmm, I think you need to use the following additional env, on exporter

```bash
PENPOT_DOMAIN_WHITE_LIST=your-domain.here.com
```

**Comment by niwinz** [comment]
In any case, this looks like obsolete configuration, and we will probably remove the need of this setting for the next release

**Comment by dploeger** [comment]
> Hmm, I think you need to use the following additional env, on exporter
> 
> ```shell
> PENPOT_DOMAIN_WHITE_LIST=your-domain.here.com
> ```

Nope. Tried that, but to no avail.

But yeah, I guess, it's some kind of leftover. Version 1.12-beta still works.

**Comment by niwinz** [comment]
Im pretty sure that in 1.14.2 it will be fixed.

**Comment by dploeger** [comment]
My issue is fixed in 1.14.1. Thanks.

**Comment by Pipazoul** [comment]
> My issue is fixed in 1.14.1. Thanks.

What do you mean by fixed ? has it been deployed on  docker hub ?, have you found a workaround ?

**Comment by dploeger** [comment]
I updated the containers to that version and it worked directly.
